### DISCUSSION TITLE
Is there a "flag" support to turn off ligatures in get_textbox( rect ) call?

### DISCUSSION THREAD

**Discussion by alexgrimmy** [pymupdf/PyMuPDF#1215] [Q&A]
This is in reference to: https://github.com/pymupdf/PyMuPDF/issues/745

Is there an equivalent of 

getText(option, flags=0)

for get_textbox(rect)?

Much thanks,
Alex

**Comment by JorjMcKie** [comment]
This method is just a short form of `get_text("text", clip=rect)`. So best fall back to this method and set the flags to your liking.
